app/cdap/components/ConfigurationGroup/PropertyRow/index.tsx

```diff
@@ -61,7 +61,7 @@ interface IPropertyRowProps extends WithStyles<typeof styles> {
   widgetProperty: IWidgetProperty;
   pluginProperty: IPluginProperty;
   value: string;
-  onChange: (values) => void;
+  onChange: (values, params?: { [key: string]: any }) => void;
```

Comment:
@lahwran @njbriggs Any better ideas for this type?

Reply:
Any reason not to pull it out as its own type with `{updateFilteredConfigurationGroups}`?